src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Android.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
```

Comment:
Does not look like to be used either

Reply:
There have been some additions to get time zone IDs, which uses `IEnumerable<string>` and `List<string>` so we now need `System.Collections.Generic`.